### PR TITLE
Add doc for the recent observation

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #578

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 30 sec

File: docs/src/user/manual/map/recent-observation.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    - GroundTruth has provided input into and coordinated large-scale, multi-national projects as well as small-scale, local projects. Through these projects, we have fostered a large and diverse client base and gained recognition for our work through several awards. 
+
+    ![Ground Truth](./img/landing-page-14.png)
+
+2. **United Nations Children's Fund (UNICEF)**: UNICEF works in over 190 countries and territories to save children's lives, defend their rights, and help them fulfil their potential, from early childhood through adolescence. And we never give up.
+
+    ![United Nations Children's Fund (UNICEF)](./img/landing-page-18.png)
+
+3. **CGIAR**: CGIAR is a global research partnership for a food-secure future dedicated to transforming food, land, and water systems in a climate crisis.CGIAR is the world’s largest global agricultural innovation network.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map/recent-observation.md]:
```suggestion
3. **CGIAR**: CGIAR is a global research partnership for a food-secure future dedicated to transforming food, land, and water systems in a climate crisis. CGIAR is the world’s largest global agricultural innovation network.
```

File: docs/src/user/manual/map/recent-observation.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# Recent Observation User Manual
+
+## Introduction:
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map/recent-observation.md:3]:
```suggestion
## Introduction:

```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 38 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 36 sec

**Pull request merged**